### PR TITLE
fix(@test): linux test run

### PR DESCRIPTION
**Pull request by alaibe**

### CODE REVIEW

**Review by noeliaSD — APPROVED** [review]
LGTM!

File: test/ui-test/src/screens/StatusWelcomeScreen.py [hunk 1]
```diff
@@ -84,7 +85,8 @@ def input_username_and_password_and_finalize_sign_up(self, username: str, passwo
         self.input_password(password)
         click_obj_by_name(SignUpComponents.FINALIZE_PASSWORD_STEP.value)
 
-        click_obj_by_name(SignUpComponents.PASSWORD_PREFERENCE.value)
+        if sys.platform == "Darwin":
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusWelcomeScreen.py:88]: Testing it, my system expects "darwin" instead of "Darwin". Please change the capital letter!

**Review by noeliaSD — CHANGES_REQUESTED** [review]

File: test/ui-test/src/screens/StatusWelcomeScreen.py (hunk 1 above)
Comment (alaibe) [reply test/ui-test/src/screens/StatusWelcomeScreen.py]: updated

**Review by noeliaSD — APPROVED** [review]
It works for me!

**Pull request merged**